assets/aws/files/bin/teleport-generate-config

```diff
@@ -561,9 +605,6 @@ EOF
     elif [[ "${USE_ACM}" == "true" ]]; then
         echo "ACM support enabled"
 
-        # set up the kubernetes listener
-        write_kubernetes_section TELEPORT_EXTERNAL_HOSTNAME
-
```

Comment:
Why is this removed here?

Reply:
@webvictim it was already written before the if section